### PR TITLE
Fixes sample config file and adds test to check it is correct

### PR DESCRIPTION
**Pull request by inetic**
[Review on Reviewable](https://reviewable.io/reviews/maidsafe/crust/270)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @vinipsmaker

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: src/config_handler.rs [hunk 1]
```diff
@@ -123,4 +123,40 @@ mod test {
...
+            Err(what) => {
+                assert!(false, format!("Error opening sample.config: {:?}", what));
+                return;
+            }
+        };
+
+        let mut encoded_contents = String::new();
+
+        match file.read_to_string(&mut encoded_contents) {
+            Ok(_) => {;},
```
Comment (vinipsmaker) [inline src/config_handler.rs]: `()` or `{}` instead weird `{;}`?

File: src/config_handler.rs [hunk 2]
```diff
@@ -123,4 +123,40 @@ mod test {
...
+            Err(what) => {
+                assert!(false, format!("Error reading sample.config: {:?}", what));
```
Comment (vinipsmaker) [inline src/config_handler.rs]: use `panic!`? seems more appropriate here (and will free you from the `return`

**Comment by vinipsmaker** [comment]
I forgot to use reviewable to do the review, sorry.

**Comment by inetic** [comment]
Thanks, fixed.

**Comment by inetic** [comment]
Np. Fixed

**Pull request merged**